docs/sources/helm-charts/tempo-distributed/get-started-helm-charts/_index.md

```diff
@@ -120,17 +136,89 @@ traces:
     enabled: false
 ```
 
-Next, we will:
+### Grafana Enterprise Traces helm chart values
 
-1. Create a `custom.yaml` file
-2. Set your storage values, the example above points to the MinIO instance configured by the chart
-3. Set your traces values to configure the receivers on the Tempo distributor
+The values in the example below provide configuration values for GET.
+
+```yaml
+---
+global:
+  clusterDomain: 'cluster.local'
+
+multitenancyEnabled: true
+enterprise:
+  enabled: true
+  image:
+    tag: v2.0.1
+enterpriseGateway:
+  enabled: true
+gateway:
+  enabled: false
+minio:
+  enabled: true
+storage:
+  trace:
+    backend: s3
+    s3:
+      access_key: 'grafana-tempo'
+      secret_key: 'supersecret'
+      bucket: 'enterprise-traces'
+      endpoint: 'tempo-minio:9000'
+      insecure: true
+  admin:
+    backend: s3
+    s3:
+      access_key_id: 'grafana-tempo'
+      secret_access_key: 'supersecret'
+      bucket_name: 'enterprise-traces-admin'
+      endpoint: 'tempo-minio:9000'
+      insecure: true
+traces:
+  otlp:
+    http:
+      enabled: true
+    grpc:
+      enabled: true
+distributor:
+  config:
+    log_received_spans:
+      enabled: true
+
+license:
```

Comment:
It looks like this snippet that embeds the license in values.yaml doesn't match the recommendations below which are a separate file or a k8s secret.  Should it be removed?

Reply:
Oh do you mean I should reorder the options below and preference the `external` secret option?